app/models/blazer/check.rb

```diff
@@ -87,7 +92,7 @@ def set_state
       end
 
       def set_nbr_of_failures
-        self.nbr_of_failures ||= -1
+        self.nbr_of_failures ||= 0
```

Comment:
I made it -1 for a reason, but forgot what that was... (no comments :see_no_evil: )